api/tests/Behat/features-v2/report-management/report-management.feature

```diff
@@ -0,0 +1,70 @@
+@v2 @report-management
+Feature: Report Management (applies to all admin roles)
+
+    @super-admin @prof-admin-completed
+    Scenario: An admin user changes report type and due date for a in progress report
+        Given a super admin user accesses the admin app
+        And a Professional Deputy has completed a Pfa Low Assets report
+        When I visit the admin client details page associated with the deputy I'm interacting with
+        And I manage the deputies 'completed' report
+        And I change the report type to 'Health and welfare'
+        And I change the report due date to '3' weeks from now
+        And I submit the new report details
+        Then the report details should be updated
+
+    @admin-manager @lay-combined-high-submitted @acs
+    Scenario: An admin user un-submits a submitted report
+        Given an admin manager user accesses the admin app
+        And a Lay Deputy has submitted a Combined High Assets report
+        When I visit the admin client details page associated with the deputy I'm interacting with
+        And I manage the deputies 'submitted' report
+        And I change the report 'start' date to '29 June 2021'
+        And I change the report 'end' date to '28 June 2022'
+        And I change the report due date to '4' weeks from now
+        And I confirm all report sections are incomplete
+        And I submit the new report details
+        Then the report details should be updated
+        When the user I'm interacting with logs in to the frontend of the app
+        Then I should see the report sections the admin ticked as incomplete labelled as changes needed
+
+    @admin @pa-admin-combined-high-submitted
+    Scenario: An admin user changes report type and due date for an un-submitted report
+        Given an admin user accesses the admin app
+        And a Public Authority Deputy has submitted a Combined High Assets report
+        When I visit the admin client details page associated with the deputy I'm interacting with
+        And I manage the deputies 'submitted' report
+        And I confirm all report sections are incomplete
+        And I submit the new report details
+        And I manage the deputies 'un-submitted' report
+        And I change the report due date to '5' weeks from now
+        And I change the report type to 'Health and welfare'
```

Comment:
Yeah so you ought to use the PA types as per above comment I believe :)

Reply:
Ah good point - I always forget that PA have a slightly different report layout.